build.sbt

```diff
@@ -46,104 +46,60 @@ lazy val panDomainAuthCore = subproject("pan-domain-auth-core")
+    libraryDependencies ++= hmacHeaders +: testDependencies
+  ).dependsOn(playSupportProject)
 
-lazy val panDomainAuthHmac_2_9 = subproject("panda-hmac-play_2-9")
-  .settings(sourceDirectory := (ThisBuild / baseDirectory).value / "pan-domain-auth-hmac" / "src")
-  .settings(
-    crossScalaVersions := Seq(scala213),
-    libraryDependencies ++= hmacLibs ++ playLibs_2_9 ++ testDependencies,
-  ).dependsOn(panDomainAuthPlay_2_9)
+lazy val panDomainAuthPlay_2_9 = playSupportFor(PlayVersion.V29)
+lazy val panDomainAuthHmac_2_9 = hmacPlayProject(PlayVersion.V29, panDomainAuthPlay_2_9)
 
-lazy val panDomainAuthHmac_3_0 = subproject("panda-hmac-play_3-0")
-  .settings(sourceDirectory := (ThisBuild / baseDirectory).value / "pan-domain-auth-hmac" / "src")
-  .settings(
-    crossScalaVersions := Seq(scala213),
-    libraryDependencies ++= hmacLibs ++ playLibs_3_0 ++ testDependencies,
-  ).dependsOn(panDomainAuthPlay_3_0)
+lazy val panDomainAuthPlay_3_0 = playSupportFor(PlayVersion.V30)
+lazy val panDomainAuthHmac_3_0 = hmacPlayProject(PlayVersion.V30, panDomainAuthPlay_3_0)
 
 lazy val exampleApp = subproject("pan-domain-auth-example")
   .enablePlugins(PlayScala)
-  .settings(libraryDependencies ++= (awsDependencies :+ ws))
-  .dependsOn(panDomainAuthPlay_2_9)
+  .dependsOn(panDomainAuthPlay_3_0)
   .settings(
-    crossScalaVersions := Seq(scala213),
+    libraryDependencies ++= awsDependencies :+ ws,
     publish / skip := true,
     playDefaultPort := 9500
   )
 
-lazy val sonatypeReleaseSettings = {
-  sonatypeSettings ++ Seq(
-    // sbt and sbt-release implement cross-building support differently. sbt does it better
-    // (it supports each subproject having different crossScalaVersions), so disable sbt-release's
-    // implementation, and do the publish step with a `+`,
-    // ie. (`releaseStepCommandAndRemaining("+publishSigned")`)
-    // See https://www.scala-sbt.org/1.x/docs/Cross-Build.html#Note+about+sbt-release
-    // Never run with "release cross" or "+release"! Odd things start happening
-    releaseCrossBuild := false,
-    releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
-    releaseProcess := Seq[ReleaseStep](
-      checkSnapshotDependencies,
-      inquireVersions,
-      runClean,
-      runTest,
-      setReleaseVersion,
-      commitReleaseVersion,
-      tagRelease,
-      setNextVersion,
-      commitNextVersion
-    )
-  )
-}
-
 lazy val root = Project("pan-domain-auth-root", file(".")).aggregate(
   panDomainAuthVerification,
   panDomainAuthCore,
-  panDomainAuthPlay_2_8,
   panDomainAuthPlay_2_9,
   panDomainAuthPlay_3_0,
-  panDomainAuthHmac_2_8,
   panDomainAuthHmac_2_9,
   panDomainAuthHmac_3_0,
   exampleApp
-).settings(sonatypeReleaseSettings)
- .settings(
-  organization := "com.gu",
+).settings(
   publish / skip := true,
+  // releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
```

Comment:
should this line be commented out?

Reply:
Unfortunately yes, due to:

https://github.com/guardian/gha-scala-library-release-workflow/issues/33